lib/static/new-ui/components/DiffViewer/index.module.css

```diff
@@ -0,0 +1,8 @@
+.image-label, .image-label + div {
```

Comment:
we need some classname for this `div`

Reply:
Problem is that we have the following structure:

```
<DiffViewer>
  <Label/>
  <SideBySIdeMode>
    <Screenshot>
      <div></div>      <—— div that we want to target
    </Screenshot>
  </SideBySIdeMode>
</DiffViewer>
```

So the target div from the react point of view is a few levels deeper than the label and we can't easily set class name in DiffViewer for that. Setting global class name inside screenshot sounds wrong too. So IMO `+ div` is the best choice out of not-so-great options.